src/entities/PoolNetwork.ts

```diff
@@ -98,14 +99,14 @@ export class PoolNetwork extends Entity {
       this._root._protocolAddresses(this.chainId).pipe(
         switchMap(({ poolManager, vaultRouter, currencies }) =>
           defer(async () => {
-            if (!currencies.length) return []
+            if (!currencies?.length) return []
```

Comment:
shouldn't be needed anymore, after checking the currencies in `_protocolAddresses()`